worksheet_D/.idea/misc.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When using PyCharm, it's best to add `.idea` to the `.gitignore` file to exclude it from the repository.